bitnami/redis/values.yaml

```diff
@@ -1959,6 +1965,36 @@ volumePermissions:
     seLinuxOptions: null
     runAsUser: 0
 
+## Kubectl InitContainer
+## used by Sentinel to update the isMaster label on the Redis(TM) pods
+##
+kubectlImage:
+  ## @param kubectlImage.command kubectl command to execute
+  ##
+  command: ["cp", "/opt/bitnami/kubectl/bin/kubectl", "/kubectl"]
+  ## @param kubectlImage.registry [default: REGISTRY_NAME] kubectl image registry
+  ##
+  registry: docker.io
+  ## @param kubectlImage.repository [default: REPOSITORY_NAME/os-shell] kubectl image repository
+  ##
+  repository: bitnami/kubectl
+  ## @skip kubectlImage.tag kubectl image tag
+  ##
+  tag: "1.29"
+  ## @param kubectlImage.pullPolicy kubectl image pull policysysctl.enabled Enable init container to modify Kernel settings
+  ##
+  pullPolicy: Always
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  ## @param kubectlImage.pullSecrets kubectl image pull secrets
+  ##
+  pullSecrets: []
```

Comment:
Could you please follow a similar schema as the one used for other images in this chart? For instance
```yaml
## @section Redis&reg; Sentinel configuration parameters
##

sentinel:
  ## @param sentinel.enabled Use Redis&reg; Sentinel on Redis&reg; pods.
  ## IMPORTANT: this will disable the master and replicas services and
  ## create a single Redis&reg; service exposing both the Redis and Sentinel ports
  ##
  enabled: false
  ## Bitnami Redis&reg; Sentinel image version
  ## ref: https://hub.docker.com/r/bitnami/redis-sentinel/tags/
  ## @param sentinel.image.registry [default: REGISTRY_NAME] Redis&reg; Sentinel image registry
  ## @param sentinel.image.repository [default: REPOSITORY_NAME/redis-sentinel] Redis&reg; Sentinel image repository
  ## @skip sentinel.image.tag Redis&reg; Sentinel image tag (immutable tags are recommended)
  ## @param sentinel.image.digest Redis&reg; Sentinel image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
  ## @param sentinel.image.pullPolicy Redis&reg; Sentinel image pull policy
  ## @param sentinel.image.pullSecrets Redis&reg; Sentinel image pull secrets
  ## @param sentinel.image.debug Enable image debug mode
  ##
  image:
    registry: docker.io
    repository: bitnami/redis-sentinel
    tag: 7.2.4-debian-11-r6
    digest: ""
```

Reply:
@carrodher  done